frontera/settings/default_settings.py

```diff
@@ -4,6 +4,7 @@
 
 AUTO_START = True
 BACKEND = 'frontera.contrib.backends.memory.FIFO'
+BACKEND_CODEC = 'frontera.contrib.backends.remote.codecs.msgpack'
```

Comment:
That looks like a setting used across backends. I would make it RedisBackend specific.